### PR TITLE
Add dummy and production SSH keys for management of k8s with Jenkins integration code

### PR DESCRIPTION
**Pull request by rtyler**
It's a little difficult to tell how well this is going to work in production,
but as far as I am aware the snippet attached is properly configuring a
credential for use by the jenkins-infra/azure Jenkinsfile and other Jenkins
Pipelines which may orchestrate k8s workloads

References [INFRA-1002](https://issues.jenkins-ci.org/browse/INFRA-1002)

### CODE REVIEW

**Comment by rtyler** [comment]
If you have some time to review @olblak, please do.

File: hieradata/common.yaml [hunk 1]
```diff
@@ -248,6 +248,38 @@ profile::pluginsite::image_tag: '24-0d1adf'
 azure::releases::account_name: 'overridewithstorageaccountname'
```
Comment (olblak) [inline hieradata/common.yaml:250]: What does 'releases' mean here?
release storage account? 
''${PREFIX}jenkinsrelease ?

**Comment by olblak** [comment]
LGTM

File: hieradata/common.yaml (hunk 1 above)
Comment (rtyler) [reply hieradata/common.yaml:250]: Yes, i'll update that with a comment and merge

**Pull request merged**